### PR TITLE
Lilygo TTGO T-OI Plus ESP32C3

### PR DESCRIPTION
**Pull request by SimonVail**

### CODE REVIEW

File: ports/espressif/boards/Lilygo-T-oi-Plus-ESP32C3/board.c [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#include "shared-bindings/microcontroller/Pin.h"
+#include "supervisor/board.h"
+
+#include "components/driver/include/driver/gpio.h"
+
+void board_init(void) {
+    
+}
```
Comment (Neradoc) [inline ports/espressif/boards/Lilygo-T-oi-Plus-ESP32C3/board.c:8]:
```suggestion
void board_init(void) {
}
```
Remove blank spaces/empty line.

File: ports/espressif/boards/Lilygo-T-oi-Plus-ESP32C3/mpconfigboard.mk [hunk 2]
```diff
@@ -0,0 +1,10 @@
+CIRCUITPY_CREATOR_ID = 0xC3C30000
+CIRCUITPY_CREATION_ID = 0x00C30001
+
+IDF_TARGET = esp32c3
+
+INTERNAL_FLASH_FILESYSTEM = 1
+
+CIRCUITPY_ESP_FLASH_MODE = dio
+CIRCUITPY_ESP_FLASH_FREQ = 80m
+CIRCUITPY_ESP_FLASH_SIZE = 4MB
```
Comment (Neradoc) [inline ports/espressif/boards/Lilygo-T-oi-Plus-ESP32C3/mpconfigboard.mk:10]:
```suggestion
CIRCUITPY_ESP_FLASH_SIZE = 4MB

```
Add newline at the end of file.

File: ports/espressif/boards/Lilygo-T-oi-Plus-ESP32C3/pins.c [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+    { MP_ROM_QSTR(MP_QSTR_LED), MP_ROM_PTR(&pin_GPIO3) },
+    { MP_ROM_QSTR(MP_QSTR_IO3), MP_ROM_PTR(&pin_GPIO3) },
+
+    { MP_ROM_QSTR(MP_QSTR_SDA), MP_ROM_PTR(&pin_GPIO19) },
+    { MP_ROM_QSTR(MP_QSTR_IO19), MP_ROM_PTR(&pin_GPIO19) },
+    { MP_ROM_QSTR(MP_QSTR_SCL), MP_ROM_PTR(&pin_GPIO18) },
+    { MP_ROM_QSTR(MP_QSTR_IO18), MP_ROM_PTR(&pin_GPIO18) },
+    
+    { MP_ROM_QSTR(MP_QSTR_IO4), MP_ROM_PTR(&pin_GPIO4) },
```
Comment (Neradoc) [inline ports/espressif/boards/Lilygo-T-oi-Plus-ESP32C3/pins.c:26]:
```suggestion
    { MP_ROM_QSTR(MP_QSTR_IO18), MP_ROM_PTR(&pin_GPIO18) },

    { MP_ROM_QSTR(MP_QSTR_IO4), MP_ROM_PTR(&pin_GPIO4) },
```
Remove blank spaces.

**Review by Neradoc — CHANGES_REQUESTED** [review]
If you accept the changes, that should make pre-commit happy.

**Comment by SimonVail** [comment]
All Check Passed !!!!

File: ports/espressif/boards/lilygo_ttgo_t-oi-plus/mpconfigboard.mk [hunk 4]
```diff
@@ -0,0 +1,10 @@
+CIRCUITPY_CREATOR_ID = 0xC3C30000
+CIRCUITPY_CREATION_ID = 0x00C30001
```
Comment (tannewt) [inline ports/espressif/boards/lilygo_ttgo_t-oi-plus/mpconfigboard.mk:2]:
```suggestion
CIRCUITPY_CREATION_ID = 0x00C30002
```

**Review by tannewt — CHANGES_REQUESTED** [review]
One small change to switch to a unique creation id. I added it here: https://github.com/creationid/lilygo-creations/blob/main/README.md

**Comment by dhalbert** [comment]
I saw the LilyGo TTO T-01-C3 we already defined, and thought maybe it is a typo, but it is a board with many fewer pins. So there 1`is a Lilygo board with `OI` (oh eye) in its anme, and another board with `01` (zero one) in its name. Oy.

**Review by dhalbert — APPROVED** [review]
Thanks for the updates!

**Pull request merged**